iosApp/iosApp/Pages/TripDetails/TripDetailsStopView.swift

```diff
@@ -26,9 +26,28 @@ struct TripDetailsStopView: View {
                         Spacer()
                         UpcomingTripView(prediction: upcomingTripViewState, routeType: routeType)
                     }
+                    .accessibilityElement(children: .combine)
+                    .accessibilityAddTraits(.isHeader)
+                    .accessibilityHeading(.h2)
                 }
             )
             scrollRoutes
+                .accessibilityLabel(scrollRoutesAccessibilityLabel)
```

Comment:
suggestion: I think this should be marked with `accessibilityElement()` as well - after the accessibility label is read, each route number is read again.

If this could be moved into the SheetNavigationLink label as well, I think that would be preferable so that it is all read together. If that is presents other challenges though then I think this is OK for now!